### ISSUE TITLE
generate kuby causes error

### ISSUE THREAD

**Issue by rept** [getkuby/kuby-core#27]
When I run this command

    bundle exec rails generate kuby

the kuby.rb gets created but I get the following:

```
create  config/initializers/kuby.rb
Traceback (most recent call last):
    19: from bin/rails:4:in `<main>'
18: from bin/rails:4:in `require'
17: from /instalane/vendor/bundle/ruby/2.5.0/gems/railties-5.2.4.3/lib/rails/commands.rb:18:in `<top (required)>'
16: from /instalane/vendor/bundle/ruby/2.5.0/gems/railties-5.2.4.3/lib/rails/command.rb:46:in `invoke'
15: from /instalane/vendor/bundle/ruby/2.5.0/gems/railties-5.2.4.3/lib/rails/command/base.rb:69:in `perform'
14: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor.rb:392:in `dispatch'
13: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor/invocation.rb:127:in `invoke_command'
12: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor/command.rb:27:in `run'
11: from /instalane/vendor/bundle/ruby/2.5.0/gems/railties-5.2.4.3/lib/rails/commands/generate/generate_command.rb:26:in `perform'
10: from /instalane/vendor/bundle/ruby/2.5.0/gems/railties-5.2.4.3/lib/rails/generators.rb:276:in `invoke'
 9: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor/base.rb:485:in `start'
 8: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor/group.rb:232:in `dispatch'
 7: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor/invocation.rb:134:in `invoke_all'
 6: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor/invocation.rb:134:in `map'
 5: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor/invocation.rb:134:in `each'
 4: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor/invocation.rb:134:in `block in invoke_all'
 3: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor/invocation.rb:127:in `invoke_command'
 2: from /instalane/vendor/bundle/ruby/2.5.0/gems/thor-1.0.1/lib/thor/command.rb:27:in `run'
 1: from /instalane/vendor/bundle/ruby/2.5.0/gems/kuby-core-0.11.4/lib/kuby/plugins/rails_app/generators/kuby.rb:24:in `create_config_file'
/instalane/vendor/bundle/ruby/2.5.0/gems/railties-5.2.4.3/lib/rails/railtie.rb:192:in `method_missing': undefined method `module_parent_name' for Instalane::Application:Class (NoMethodError)
```

I suspect it has something to with the version of thor (which I cannot update because of another dependency).

I installed kuby-core (0.11.4) and kuby-digitalocean (0.4.2).  Thor is version 1.0.1 like you can see from the error.

Solution might be to update minimum version required in the Gemspec of thor?

Side question: should kuby be installed in the development group of the Gemfile?  Probably not since you mentioned elsewhere that there is middleware that responds to health checks.  Does this impact memory / performance while running in production?  Or could it be a plan to split up the gem so most part goes into development except for the health check?

**Comment by camertron** [comment]
Hey @rept, it looks like I introduced this bug by using a method called `module_parent_name` that was introduced in Rails 6. Looks like you're using Rails 5.2, which explains the error. I just published v0.11.6 that calls the right method for the current Rails version.

**Comment by rept** [comment]
Hi @camertron,

Thanks.  Tested again, and works now!  Will try to play around with it this weekend.
